lib/server.js

```diff
@@ -43,19 +43,21 @@ module.exports = async (req, res, flags, current, ignoredFiles) => {
 
   related = decodeURIComponent(path.format(related))
 
+  const relatedExists = fs.existsSync(related)
```

Comment:
Can you do `await fs.exists()` please? Not sure if that will throw or not. Wrap it in `try` if it will... 😊